src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/EmbeddedLanguages/VirtualChars/CSharpVirtualCharService.cs

```diff
@@ -120,15 +136,15 @@ private static VirtualCharSequence TryConvertRawStringToVirtualChars(SyntaxToken
             var startIndexInclusive = 0;
             var endIndexExclusive = tokenText.Length;
 
-            if (IsAnyRawStringLiteralToken(token))
+            if (token.Kind() is SyntaxKind.UTF8SingleLineRawStringLiteralToken)
```

Comment:
💡 `IsKind` is faster than `Kind` for single checks. We haven't been able to _prove_ this makes a difference in practice, but I expect to eventually be able to turn on this analyzer.

```suggestion
            if (token.IsKind(SyntaxKind.UTF8SingleLineRawStringLiteralToken))
```

Reply:
so here's the thing.  i prefer the `is` check because it's more in line with the language and scales up naturally to using things like `is X or Y`.  absent a strong need, i'd just prefer that we ignore this and let individual dev decide what they prefer.